recipe/meta.yaml

```diff
@@ -21,10 +21,13 @@ requirements:
     - cmake
     - make                                           # [unix]
   host:
-    - msinttypes                                     # [win and vc<14]
-    - zlib
-    - lz4-c                                          # [unix]
-    - zstd
+    # we probably won't ever use vc<14 at this point, but this is left here just in case
+    - msinttypes =r26                                # [win and vc<14]
+    # pinnings below are taken from upstream's internal-complibs folder.
+    # zlib's patch version had to be bumped up by one to satisfy zstd's dependency requirements on it.
+    - zlib =1.2.12
+    - lz4-c =1.9.4
+    - zstd =1.5.2
```

Comment:
`zstd` is already **1.5.2** in `cbc.yaml`, see https://github.com/AnacondaRecipes/aggregate/blob/331d0d2487f7d74cc929e1f0a630a5fa26498e27/conda_build_config.yaml#L230-L231. So just `zstd` will be enough

Reply:
thanks, you're right, see note below though